src/style.css

```diff
@@ -1235,7 +1235,7 @@ blockquote p:before {
 .rule-content .china {
```

Comment:
Can we merge this with `.rule-content .info` ? 
`.rule-content .info, .rule-content .china {`